src/logic.js

```diff
@@ -392,10 +435,9 @@ define([
                 }
             }, _this = this;
 
-            ret.preload = _.chain(this.all)
-                .filter(function(ref) {
-                    return isCaseReference(ref.path);
-                })
+            ret.preload = _.chain(this.reverse[CASE_REF_ID] || {})
+                .values()
+                .flatten(true)
                 .map(function(ref) {
                     var info = ref.path.split('/'),
                         prop = info[2];
```

Comment:
@emord: is this still correct now that case references look like `#case/property` instead of `#case/type/property`? Do we want something more like (python pseudo code) `ref.path.split("/", 1)[1]`?